### DISCUSSION TITLE
VulnerabilityAlerts for Golang is not available

### DISCUSSION THREAD

**Discussion by MokkeMeguru** [renovatebot/renovate#18726] [Request Help] [answered]
Hi.

I now setup renovate vulnerabilityAlerts settings on the repository https://github.com/MokkeMeguru/go-renovate-security-test
I want to labeled the golang security updates. 
However, JavaScript's security update is labeled as security, but Golang's security update is not labeled. 

Does vulnerabilityAlerts support only some languages ? If so that, is there the list of support language of vulnerability ?

Below is my configuration of renovate. 

```json
{
  "vulnerabilityAlerts": {
    "labels": ["security"],
    "commitMessageSuffix": "[SECURITY]",
    "prCreation": "immediate"
  }
}
```

Thank you.

Note: 
When I found the renovate's log, Renovate found the alerted dependency from GitHub Correctly. (https://app.renovatebot.com/dashboard#github/MokkeMeguru/go-renovate-security-test/874424351)

```text
DEBUG: GitHub vulnerability details
{
  "alerts": {
    "go/github.com/labstack/echo/v4": {
      "< 4.9.0": "4.9.0"
    },
    "npm/d3-color": {
      "< 3.1.0": "3.1.0"
    }
  }
}
```

**Comment by rarkins** [comment, marked as answer]
This functionality has been added to Renovate and will be live in the app within the next day

**Reply by MokkeMeguru** [reply]
good news. I wait for the renovate's new release. thank you.
